app/feature/feature-odyssey/src/main/kotlin/com/hedvig/android/feature/odyssey/step/summary/ClaimSummaryDestination.kt

```diff
@@ -126,6 +256,13 @@ private fun ClaimSummaryScreen(
   }
 }
 
+private fun getIconFromMimeType(mimeType: String) = when (mimeType) {
+  "image/jpg" -> Icons.Hedvig.Pictures
+  "video/quicktime" -> Icons.Hedvig.Play
+  "application/pdf" -> Icons.Hedvig.Document
+  else -> Icons.Hedvig.Document
+}
```

Comment:
Using the existing component will also help us by avoiding creating this function a third time too.
I though we only had it in one place, I now realize we got it in 2 already, but I think that's another thing that we want to fix. It should be possible to have 1 unified way to render this that we can re-use from the now 3 places we want to show it.

Reply:
Ok! I'll move it to core-icons then.

Reply:
Hmm I had something else in mind actually here.
core-icons is a nice little module that can contain all of the `Icons.Hedvig.Foo` icons in there, I think we should keep it like that actually. Moving it there will mean that basically every single feature module that we have, since it needs to access our icons then automatically gets access to the `getIconFromMimeType` but they really do not need to know about it. I would optimally prefer this to be a private function inside the UI component which is made to render the grid with those icons.

So this `FilesLazyVerticalGrid` component can stay in core-ui, since this is a common UI component that we use througout our app, but it's not necessarily part of our design-system, it's a bit bigger than that, hence core-ui is the right place for it to be. And this `getIconFromMimeType` can stay private in there I think.

Reply:
Ok, got it! Moved it here, though still left it public because ClaimDetailsDestination also uses it outside FilesLazyVerticalGrid.